dapp_runner/runner/service.py

```diff
@@ -100,7 +100,7 @@ async def get_service(
     try:
         payload_instance = payloads[desc.payload]
     except KeyError:
-        raise RunnerError(f"Undefined payload: `{desc.payload}`")
+        raise RunnerError(f'Undefined payload: "{desc.payload}"')
```

Comment:
noted, _but_ still - I'd say this would be resolved by adding more granular validation to the descriptor parser -> once the descriptor is loaded without errors, any errors happening in the Runner are supposed to raise `RunnerError`...